assets/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "module": "es6",
     "moduleResolution": "node",
     "noImplicitAny": true,
+    "skipLibCheck": true,
```

Comment:
This is standard in TypeScript projects initialized with `tsc --init` and now we don't have to remember to include it as a CLI arg.